src/lbaf/Applications/LBAF_app.py

```diff
@@ -371,6 +371,10 @@ def run(self):
             self.__logger.info("No brute force optimization performed")
             a_min_max = []
 
+        # Perform central prefix optimizer when requested
+        if self.__parameters.algorithm["name"] != "CentralizedPrefixOptimizer":
```

Comment:
I don't understand why logging `Starting centralized prefix optimizer` if the algorithm is **not** the `CentralizedPrefixOptimizer`

Reply:
Removed this logging line which in any event indicates the opposite of what it means